SeQuant/domain/mbpt/models/cc.cpp

```diff
@@ -292,36 +292,34 @@ std::vector<ExprPtr> CC::eom_r(size_t nann, size_t ncre) {
 
   // initialize result vector
   std::vector<ExprPtr> result;
-  auto idx = std::max(nann, ncre);  // index for populating the result vector
+  using std::max;
+  auto idx = max(na, nc);  // index for populating the result vector
   result.resize(idx + 1);
 
-  using boost::numeric_cast;
   // start from the highest excitation order, go down to the lowest possible
-  for (auto na = numeric_cast<std::int64_t>(nann),
-            nc = numeric_cast<std::int64_t>(ncre);
-       na > 0 || nc > 0; --na, --nc) {
+  for (std::int64_t ra = na, rc = nc; ra > 0 || rc > 0; --ra, --rc) {
     // project with <ncre, nann| (i.e., multiply P(ncre, nann)) and compute VEV
-    result.at(idx) = vac_av(P(nc, na) * hbar_R, op_connect);
+    result.at(idx) = vac_av(P(nₚ(ra), nₕ(rc)) * hbar_R, op_connect);
```

Comment:
@ajay-mk @bimalgaudel @cmasteran1 @Krzmbrzl comments welcome on such use of unicode (which avoids overly-short `np` and `nh`)